### ISSUE TITLE
Focal points in Xhtml

### ISSUE THREAD

**Issue by jon-anders** [ErikHen/PictureRenderer.Optimizely#14]
In PictureRenderer.Optimizely, Version=2.5.0.0
When calling RenderImageAsPicture() focal points are ignored.

GetPictureFromImg() returns Picture.Render(... focalPoint: default((double, double)) ...) and does not check if the image has a focal point set.

**Comment by ErikHen** [comment]
That is true. The picture data is fetched from the img element that TinyMCE creates, and that does not contain any focal point data. 
It might be possible to get the actual IContent for the image from the Url ([https://world.optimizely.com/csclasslibraries/cms/EPiServer.Web.Routing.UrlResolver?version=12#EPiServer_Web_Routing_UrlResolver_Route_EPiServer_UrlBuilder_](https://world.optimizely.com/csclasslibraries/cms/EPiServer.Web.Routing.UrlResolver?version=12#EPiServer_Web_Routing_UrlResolver_Route_EPiServer_UrlBuilder_)).
It would mean a performance hit, but might be acceptable.  

It will be some time before I'm able to look into this, unfortunately.
